online-store/src/components/data/data.ts

```diff
@@ -0,0 +1,225 @@
+export interface IItem {
+  id: number;
+  name: string;
+  quantity: number;
+  year: number;
+  manufacturer: string;
+  color: string;
+  cams: number;
+  favorite: boolean;
+}
+
+export const data: IItem[] = [
+  {
+    id: 1,
```

Comment:
`src`: ??? Usually backend sends you data already with src, so you should not calculate it from the 'id'